build.gradle

```diff
@@ -395,7 +395,7 @@ dependencies {
     integrationTestImplementation group: 'net.serenity-bdd', name: 'serenity-spring', version: versions.serenity
     integrationTestImplementation group: 'io.rest-assured', name: 'rest-assured', version: versions.rest_assured
 
-    pitest 'org.pitest:pitest-junit5-plugin:1.7.4'
+    pitest group: 'org.pitest', name: 'pitest-junit5-plugin', version:versions.pitest
```

Comment:
Please update 
pitest group: 'org.pitest', name: 'pitest-junit5-plugin', version:versions.pitest
to 
pitest group: 'org.pitest', name: 'pitest-junit5-plugin'

as this is already on higher version. So simply remove version, then it will take suitable version

Reply:
Removed version.